sirepo/template/srw.py

```diff
@@ -1772,6 +1772,7 @@ def _process_image(data):
         s = srwl_uti_smp.SRWLUtiSmp(
             file_path=path,
             area=None if not int(m['cropArea']) else (m['areaXStart'], m['areaXEnd'], m['areaYStart'], m['areaYEnd']),
+            extTr=int(m['transmissionImage']),
```

Comment:
The extTr field doesn't exist on the srwl_uti_smp.SRWLUtiSmp module - this is used only when previewing the sample by selecting the "Download Processed Image" (the filter icon) next to the sample file name. I don't think the extTr value applies in that case.

![Screen Shot 2019-07-26 at 11 03 13 AM](https://user-images.githubusercontent.com/6516910/61968328-fca1e600-af94-11e9-830c-4c32b571a8bc.png)

Reply:
yes! you are correct! thanks for your revision. :)